box_embeddings/parameterizations/tanh_activated_box_tensor.py

```diff
@@ -0,0 +1,147 @@
+"""
+    Implementation of sigmoid box parameterization.
+"""
+from typing import List, Tuple, Union, Dict, Any, Optional, Type
+from box_embeddings.parameterizations.box_tensor import (
+    BoxTensor,
+    BoxFactory,
+    TBoxTensor,
+)
+import box_embeddings.common.constant as constant
+import torch
+import warnings
+
+
+@BoxFactory.register_box_class("tanh_activated")
+class TanhActivatedBoxTensor(BoxTensor):
+
+    """
+    Tanh Activated Box Tensor
+    """
+
+    def __init__(
+        self, data: Union[torch.Tensor, Tuple[torch.Tensor, torch.Tensor]]
+    ):
+        """
+
+        Args:
+            data: The weights for the box
+        """
+        super().__init__(data)
+        self.tanh_eps = 1e-20
```

Comment:
Are we using this or the value coming from the constant file?

Reply:
Removed this, value is coming from the constant file.